manager.go

```diff
@@ -0,0 +1,220 @@
+package sandbox
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/bmizerany/pat"
+	"github.com/dchest/uniuri"
+	"gopkg.in/vinxi/layer.v0"
+	"gopkg.in/vinxi/vinxi.v0"
+)
+
+type Layer struct {
+	layer *layer.Layer
+}
+
+type Options struct {
+	Optional bool
+}
+
+type Rule interface {
+	ID() string
+	Name() string
+	Description() string
+	// Options() Options
+	JSONConfig() string
+	Match(*http.Request) bool
+}
+
+type Scope struct {
+	disabled bool
+	rules    []Rule
+	plugins  *PluginLayer
+
+	ID          string
+	Name        string
+	Description string
+}
+
+func NewScope(rules ...Rule) *Scope {
+	return &Scope{ID: uniuri.New(), Name: "default", plugins: NewPluginLayer(), rules: rules}
```

Comment:
If you create two scopes, you would have two `"default"` scopes, right? Would that make sense?